tests/device_tests/common.py

```diff
@@ -35,6 +35,18 @@
     "gesture negative beard leaf deadline stadium vegan employer armed marathon alien lunar broken edge justice military endorse diet sweater either",
     "gesture negative beard lily desert belong speak realize explain bolt diet believe response counter medal luck wits glance remove ending",
 ]
+MNEMONIC_SUPERSHAMIR_128BIT = [
```

Comment:
The const above is called `MNEMONIC_SHAMIR_20..`, the 20 stands for 20 words. We might stick to that? Also let's rename the Shamir/SuperShamir to Slip39_Basic and Slip39_Advanced?

Reply:
And let's rename the test files as well?

Reply:
so MNEMONIC_SLIP39_ADVANCED_20/33? ok will rename and I'll rename the files too.

Reply:
Sounds good to me.

Reply:
Renamed everything. Based on discussion on slack I added bip39 to test names where relevant. I didn't rename common.MNEMONIC* values for bip39 because it would touch on too many files and it doesn't seem necessary.